ts/features/idpay/initiative/configuration/xstate/actions.ts

```diff
@@ -9,11 +9,23 @@ import { showToast } from "../../../../../utils/showToast";
 import { IDPayDetailsRoutes } from "../../details/navigation";
 import { IDPayConfigurationRoutes } from "../navigation/navigator";
 import { Context } from "./context";
+import { Events } from "./events";
+
+const skipNavigation = (event: Events) => {
+  if (event.type === "BACK") {
+    return event.skipNavigation;
+  }
+  return false;
+};
```

Comment:
```suggestion
const skipNavigation = (event: Events) =>
  event.type === "BACK" ? event.skipNavigation : false;
```

Reply:
Yes, that's better. I'm also working on a wrapper function to facilitate the `skipNavigation` check on actions.

Reply:
In https://github.com/pagopa/io-app/pull/4496/commits/77c3257fbc6dfb200439ec9d3deb06143c121721 I have added a global event, `GLOBAL_E_BACK`, and a common utility function `guardedNavigationAction`, that has the purpose of checking if the action should be executed or not based on the received event.